### PR TITLE
feat(app): never hide module/liquid setup

### PR DESCRIPTION
**Pull request by jerader**
closes RAUT-570 RAUT-574

# Overview

Now, when you have a protocol with no liquids and no modules in the desktop app, there are 2 things:
1) the liquids tab always shows in protocol details and shows the empty state
2) the module and liquid steps in protocol setup are shown but are in a disabled state

# Test Plan

Make sure that those sections mentioned work as expected in the desktop app

# Changelog

- created an `EmptySetupStep` component and test for the modules and liquids
- added an empty state to the liquid protocol details
- added test coverage

# Review requests

see test plan

# Risk assessment

low

### CODE REVIEW

File: app/src/organisms/ProtocolDetails/ProtocolLiquidsDetails.tsx [hunk 1]
```diff
@@ -34,22 +45,45 @@ export const ProtocolLiquidsDetails = (
...
+                  labwareByLiquidId={labwareByLiquidId}
+                />
+              </Flex>
+              {index < liquidsInLoadOrder.length - 1 && <Divider />}
+            </React.Fragment>
+          )
+        })
+      ) : (
+        <Flex
+          padding={` ${SPACING.spacing16} ${SPACING.spacing16} ${SPACING.spacing32} ${SPACING.spacing16}`}
```
Comment (koji) [inline app/src/organisms/ProtocolDetails/ProtocolLiquidsDetails.tsx:70]:
```suggestion
          padding={`${SPACING.spacing32} ${SPACING.spacing16}`}
```

File: app/src/organisms/ProtocolDetails/ProtocolLiquidsDetails.tsx [hunk 2]
```diff
@@ -34,22 +45,45 @@ export const ProtocolLiquidsDetails = (
...
+          textAlign={TYPOGRAPHY.textAlignCenter}
+          gridGap={SPACING.spacing12}
+          flexDirection={DIRECTION_COLUMN}
+        >
+          <Icon
+            color={COLORS.medGreyEnabled}
+            alignSelf={ALIGN_CENTER}
+            size="20px"
```
Comment (koji) [inline app/src/organisms/ProtocolDetails/ProtocolLiquidsDetails.tsx]:
```suggestion
            size="1.25rem"
```

File: app/src/organisms/ProtocolDetails/index.tsx [hunk 3]
```diff
@@ -569,20 +566,19 @@ export function ProtocolDetails(
...
                   >
-                    <StyledText
-                      textTransform={TYPOGRAPHY.textTransformCapitalize}
-                    >
-                      {t('liquids')}
-                    </StyledText>
-                  </RoundTab>
-                )}
+                    {t('liquids')}
+                  </StyledText>
```
Comment (koji) [inline app/src/organisms/ProtocolDetails/index.tsx]:
```ts
<StyledText>
   {i18n.format(t('liquids'), 'capitalize')}
</StyledText>
```

Also can we update other tabs?

File: app/src/organisms/ProtocolDetails/ProtocolLiquidsDetails.tsx (hunk 1 above)
Comment (koji) [inline app/src/organisms/ProtocolDetails/ProtocolLiquidsDetails.tsx]: ProtocolDetails(L594) has the following padding so for this part, we don't need padding for left and right.
```ts
              padding={`${SPACING.spacing16} ${SPACING.spacing16} 0 ${SPACING.spacing16}`}

```

```suggestion
          paddingTop={SPACING.spacing16}
          paddingBottom={SPACING.spacing32}
```

Comment (koji) [reply app/src/organisms/ProtocolDetails/ProtocolLiquidsDetails.tsx]: https://www.figma.com/file/0hYQ4lFJbAAI33jQMRt1Di?node-id=1836:44526&mode=dev#502945950

**Review by koji — APPROVED** [review]
Left a few comments but EmptySetupStep and No Liquid case worked as expected.

File: app/src/organisms/ProtocolDetails/index.tsx (hunk 3 above)
Comment (jerader) [reply app/src/organisms/ProtocolDetails/index.tsx]: yup!

**Pull request merged**